PicPi/AddPhotoViewController.swift

```diff
@@ -90,11 +89,16 @@ class AddPhotoViewController: UIViewController {
   
   // MARK: - Buttons Actions
   @objc func sendButtonPressed(sender: UIButton!) {
+    /// testing FramAPI using photoUploadManager
     photoUploadManager.startUploadPreviouslySelectedPhotos()
-    sendButton.isEnabled = false
-    
+    //    sendButton.isEnabled = false
+    photoUploadManager.getThumbnails()
+    photoUploadManager.updatePref(preferences: ["name" : "TotoPI" , "music" : false , "slideshowDuration" : 454])
   }
   @objc func doneButtonPressed(sender: UIButton!) {
+    /// testing FramAPI using photoUploadManager
+    photoUploadManager.retrievePIPrefrences()
+    photoUploadManager.removePhoto(filename: "IMG_4449.PNG")
     self.dismiss(animated: true, completion: nil)
```

Comment:
nit: these changes are for testing right? If so, mind removing them? They might cause unnecessary merge conflicts

Reply:
Yes!
Removed! :)